### PR TITLE
Avoid race condition

### LINKED ISSUE #20: How to avoid race condition
Thank you for the awesome OSS every time!
There was one point I wanted to check while using it in my business.

## Overview

Is there any way to avoid race condition?

## What steps will reproduce the race condition?
```
go version go1.17.5 darwin/amd64
```
1. Checkout [this commit](https://github.com/mattn/go-gimei/pull/19/commits/229c424a606866bbc3e49f4c7666515ea9fb4b6d).
2. Execute `go test ./... -race`.
3. Get a race condition warning.

## Additional information
If it's not supported, I'd like to implement it.

### PR DESCRIPTION
**Pull request by yagipy**
I'm sure you're busy, but please check it out when you get a chance.🙏

## Overview
Use sync.Mutex to avoid race condition warnings.

closes #20

### CODE REVIEW

File: gimei.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ var (
 	onceName    sync.Once
 	onceAddress sync.Once
 	r           *rand.Rand
+	mu          = &sync.Mutex{}
```
Comment (mattn) [inline gimei.go:23]: Just use `mu sync.Mutex`

Comment (yagipy) [reply gimei.go]: Thanks for review:)
Fixed it.

**Comment by yagipy** [comment]
@mattn 
I fixed your review, so please check it out when you get a chance.🙏

**Review by mattn — APPROVED** [review]

**Comment by mattn** [comment]
Thank you

**Pull request merged**